pixel_clusterizer/clusterizer.py

```diff
@@ -9,11 +9,25 @@
                           ('charge', '<u2')])
 
 
+def np_uint_type_chooser(number):
```

Comment:
What do you mean with space and time tradeoff? The function is used to reduce memory footprint and is only called when the array size changes. That is in total just a few functions calls.